diff --git a/src/ERC20.sol b/src/ERC20.sol
index 77b2fbc..6a47fb2 100644
--- a/src/ERC20.sol
+++ b/src/ERC20.sol
@@ -191,7 +191,7 @@ abstract contract ERC20 {
 
   function _burn(address from, uint256 amount) internal virtual {
     _beforeTokenTransfer(from, address(0), amount);
-    balanceOf[from] -= amount;
+    //balanceOf[from] -= amount;
 
     // Cannot underflow because a user's balance
     // will never be larger than the total supply.
